tests/fixtures.py

```diff
@@ -1,959 +0,0 @@
-# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
```

Comment:
Wow.

Reply:
As expected since `tests/fixtures.py` is legacy code that should not be used